lte/gateway/c/oai/tasks/nas/esm/DedicatedEpsBearerContextActivation.c

```diff
@@ -152,7 +153,8 @@ int esm_proc_dedicated_eps_bearer_context(
       mbr_dl,
       mbr_ul,
       tft,
-      pco);
+      pco,
+      gtp_teid);
```

Comment:
Not needed.

Reply:
Bearer context for the new dedicated bearer gets created at NAS similar to default bearer context. Hence gtp_teid is sent from MME to NAS. Once the bearer context is created, gtp_teid is stored in the context by NAS and MME fetches it from the context to send it in ERAB SETUP REQ to eNB

Reply:
NAS shall not know anything about TE id. Please change it , Lets discuss in next call.